frontend/src/app/App.test.tsx

```diff
@@ -1263,14 +1221,16 @@ describe("App.onHistoryChange", () => {
     expect(instance.state.currentPageScriptHash).toEqual("sub_hash")
 
     window.history.back()
-    await waitFor(() =>
+    waitFor(() => {
```

Comment:
It looks like `waitFor` returns a promise, so not awaiting it means that we're actually not testing the inner assertions, I think? I might be misunderstanding, but it looks like removing the `await` here is not what we want. Can we confirm?

Reply:
Correct, docs show correct usage with `await` since it returns a promise 👍🏼 
Not sure what I changed thats causing this test to fail, but will keep trying.